draft-ietf-lpwan-schc-over-lorawan.md

```diff
@@ -81,7 +81,8 @@ and modes of operation need to be set appropriately for each of the LPWAN
 technologies.
 
 This document describes the parameters and modes of operation when
-SCHC is used over LoRaWAN networks.
+SCHC is used over LoRaWAN networks. LoRaWAN protocol is specified by the
+LoRa® Alliance in {{lora-alliance-spec}}
```

Comment:
LoRa Alliance should be noted `LoRa Alliance®` 
See https://lora-alliance.org/ for public references

Reply:
Not sure RFCs need to follow those (r) and (tm) rules though

Reply:
Thanks, corrected. I checked with Olivier S. and he advised me to put the registered for the first apparition of LoRa, LoRaWAN and LoRa Alliance. Not sure about IETF rule though, I'll ask the editors.